packages/components/ng-ovh-contracts/src/template.html

```diff
@@ -0,0 +1,101 @@
+<div class="contracts-ctnr">
+
+    <div data-ng-if="ContractsCtrl.fullText === false">
+        <table class="table table-bordered"
+               data-ng-class="{
+                   'table-striped': ContractsCtrl.ovhContracts.length > 1
+               }">
+            <tr data-ng-repeat="contract in ContractsCtrl.ovhContracts track by $index">
+                <td>
+                    <a target="_blank"
+                       rel="noopener"
+                       data-ng-href="{{ contract.url || contract.pdf }}">
+                        <i class="fa fa-file-pdf-o" aria-hidden="true"></i>
+                        <span data-ng-bind="contract.name"></span>
+                    </a>
+                </td>
+            </tr>
+        </table>
+        <div class="checkbox">
+            <label>
+                <input type="checkbox"
+                       data-ng-model="ContractsCtrl.agree" />
+                <span data-ng-if="ContractsCtrl.ovhContracts.length > 1"
+                      data-translate="ovh_contracts_agree">
+                </span>
+                <span data-ng-if="ContractsCtrl.ovhContracts.length === 1"
+                      data-translate="ovh_contract_agree">
+                </span>
+            </label>
+        </div>
+    </div>
+
+    <div data-ng-if="ContractsCtrl.fullText === true">
+        <p data-ng-if="ContractsCtrl.ovhContracts.length > 1"
+           data-translate="ovh_contracts_list_title"
+           data-translate-values="{ 'nbr': ContractsCtrl.ovhContracts.length }">
```

Comment:
```suggestion
           data-translate-values="{ 'number': ContractsCtrl.ovhContracts.length }">
```
is that really harder to write ? 😄